src/test/ui/consts/const-points-to-static.rs

```diff
@@ -3,8 +3,9 @@
 #![allow(dead_code)]
 
 const TEST: &u8 = &MY_STATIC;
-//~^ skipping const checks
-//~| it is undefined behavior to use this value
+//~^ ERROR it is undefined behavior to use this value
+//~| NOTE  encountered a reference pointing to a static variable
```

Comment:
Another curious thing about diagnostic UI tests: looks like `NOTE` annotations match what the rustc API calls "labels". Do they also match what the rustc API calls "note"?